py/kubeflow/kfctl/testing/pytests/conftest.py

```diff
@@ -1,10 +1,6 @@
 import pytest
 
 def pytest_addoption(parser):
-  parser.addoption(
```

Comment:
I don't know if vendor deploy this in their GCP environment. I think it should be safe since GCP users also moves to other tools